packages/payload/src/collections/config/types.ts

```diff
@@ -26,6 +32,7 @@ import type {
   LivePreviewConfig,
   MetaConfig,
   PayloadComponent,
+  AfterErrorHookArgs as RootAfterErrorHookArgs,
```

Comment:
Any specific reason for this rename?

Reply:
I wanted to clarify that it's not for `CollectionAfterErrorHook` but I don't think it's needed, changed